README.md

```diff
@@ -36,6 +36,33 @@ To see the graphics, you can open another terminal and run
 GAZEBO_MODEL_PATH=`pwd`/../python/social_bot/models gzclient
 ```
 
+## Environments and Tasks
+
+### [Simple Navigation](python/social_bot/envs/simple_navigation.py)
```

Comment:
can you also add some description about these tasks?

Reply:
ok